testutil-gcloud/src/main/java/io/spine/testing/server/storage/datastore/EmulatorContainer.java

```diff
@@ -0,0 +1,107 @@
+package io.spine.testing.server.storage.datastore;
+
+import com.google.cloud.NoCredentials;
+import com.google.cloud.ServiceOptions;
+import com.google.cloud.datastore.DatastoreOptions;
+import com.google.errorprone.annotations.CanIgnoreReturnValue;
+import io.spine.server.storage.datastore.ProjectId;
+import org.checkerframework.checker.nullness.qual.MonotonicNonNull;
+import org.testcontainers.containers.DatastoreEmulatorContainer;
+import org.testcontainers.utility.DockerImageName;
+
+import static java.util.Objects.requireNonNull;
+
+/**
+ * A container running the Emulator image.
+ *
+ * <p>The container is started in its "full consistency" mode.
+ */
+class EmulatorContainer extends DatastoreEmulatorContainer {
```

Comment:
Can we have this class `final`, if we do not want to extend it in the same package?

Reply:
Addressed.